lxd/instance/drivers/driver_qemu_templates.go

```diff
@@ -223,6 +224,7 @@ mount_tag = "{{.mountTag}}"
 `))
 
 // Devices use "lxd_" prefix indicating that this is a user named device.
+// The device name prefix must not be changed as it used by MAAS to identify devices inside the VM.
```

Comment:
This comment is only true in this case. It's not exposed to the VM outside of disk devices, so the others we could change.

I'd also change the comment to be more generic and say that we want to have /dev/disk/by-id be a usable stable identifier.

Reply:
@stgraber I've updated that thanks